package.json

```diff
@@ -36,18 +32,6 @@
       "eslint --fix --no-ignore --max-warnings=0 -c .eslintrc.cjs"
     ]
   },
-  "browserslist": {
```

Comment:
nice. thanks.
for any readers of this PR, this is removed for [this](https://github.com/codecov/gazebo/pull/3347#discussion_r1786618925)